worker-languagedetection/README.md

```diff
@@ -14,7 +16,11 @@ Worker configuration is supported through the following environment variables:
 
  - `WORKER_LANG_DETECT_SOURCE_FIELD`  
     Default: `CONTENT`  
-    Used to specify which document field is used for data sources
+    Used to specify which document field is used for data sources, however if fieldSpecs is passed as custom data this value is ignored.
+
+-  `WORKER_LANG_DETECT_FIELD_PREFIX`
+	Default: `LANGUAGE_CODE_`
+    This is used when processing using the new mode of operation that can process multiple fields at the same time. It will specify the name of the field that is to be added to the document to record results. For example if the worker is checking the `CONTENT` field it will add the field `LANGUAGE_CODE_CONTENT` to the document.
```

Comment:
Say "Multi-Field mode of operation" rather than referring to "the new mode of operation", which would just add a statement that will need removed or changed in a subsequent release when it's no longer new.

"It will specify the name of the field that is to be added..." - Make it clear that this is a prefix, not the name of the field.